### PR TITLE
fbook add slingshot

### PR DESCRIPTION
**Pull request by Thchoonlophon**

### CODE REVIEW

File: participants/fbook.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+## (8) What pre-processing are you doing before ingesting the data?
+
+- Import via PushPath, additional upload interface for `texture hub/bucket` technology
+- Each file is between 1Mib-10Mib
+- File data information directory path data or table partition information will be stored in ipfs to retrieve specific data by cid, name and time
+
+## (9)  What tech stack will you use to store the data?
+
+*Either: Powergate, Hosted Powergate, Textile Hub/Buckets, lotus, or some other solution. Tell us what you're planning to use and how.*
+
```
Comment (dineshshenoy) [inline participants/fbook.md:51]: Can you clarify what stack you will be using please?

File: participants/fbook.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- Provide downloads of novels
+- search with cid
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+**txt**
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+**free ebook**
```
Comment (dineshshenoy) [inline participants/fbook.md:32]: Can you clarify the source of these books and whether you have the appropriate license?

**Comment by Thchoonlophon** [comment]
@dineshshenoy

File: participants/fbook.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+- search with cid
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+**not curated dataset**
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+**some Public Domain books**
+**Or which Copyright Expiration**
```
Comment (dineshshenoy) [inline participants/fbook.md:33]: I'm still not clear where your books are coming from.  Is this from a website that has public domain books?

Comment (guowenshuai) [reply participants/fbook.md:33]: http://www.feedbooks.com/publicdomain
https://www.gutenberg.org/
and so on.

**Pull request merged**